Lib/configparser.py

```diff
@@ -1057,7 +1089,7 @@ def _read(self, fp, fpname):
                         elements_added.add(sectname)
                     # So sections can't start with a continuation line
                     optname = None
-                # no section header in the file?
+                # no section header?
                 elif cursect is None:
                     raise MissingSectionHeaderError(fpname, lineno, line)
                 # an option line?
```

Comment:
Since this will always create an `UNNAMED_SECTION`, regardless of whether it's actually used or not, I propose to delete that section near the end of the method in case it's empty. This can be done by inserting before `self._join_multiline_values()` (line 1122):

```python3
        # Delete UNNAMED_SECTION if it's empty
        if (UNNAMED_SECTION in self._sections and not
                self._sections[UNNAMED_SECTION]):
            del self._sections[UNNAMED_SECTION]
            del self._proxies[UNNAMED_SECTION]
```

Reply:
You're a welcome to commit on this PR. I'll do this time.